commander/src/bootstrapping/commands/config/create.ts

```diff
@@ -81,17 +81,14 @@ export class CreateCommand extends Command {
 			if (!userResponse.confirm) {
 				this.error('Operation cancelled, config file already present at the desired location');
 			} else {
-				fs.writeJSONSync(resolve(configPath, 'config.json'), defaultConfig, {
-					spaces: '\t',
-					mode: OWNER_READ_WRITE,
-				});
+				const res = await handleOutputFlag(configPath, defaultConfig, 'config', 'config.json');
+				this.log(res);
 			}
 		} else {
 			fs.mkdirSync(configPath, { recursive: true });
-			fs.writeJSONSync(resolve(configPath, 'config.json'), defaultConfig, {
-				spaces: '\t',
-				mode: OWNER_READ_WRITE,
-			});
+
+			const res = await handleOutputFlag(configPath, defaultConfig, 'config', 'config.json');
```

Comment:
```suggestion
			const res = await handleOutputFlag(configPath, defaultConfig, 'config', 'config');
```

sometimes we are passing with extension and sometimes we don't. Maybe to use one standard

Reply:
The filename is optional. If not present then the namespace is used. But yes, we can omit the filename unless necessary.